.tours/1---welcome.tour

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "1 - Welcome to Easy Spaces Sample App",
+    "steps": [
+        {
+            "file": "sfdx-project.json",
+            "description": "The app uses a modular application design pattern. The app comprises four different unlocked packages, and the source code and metadata components for each package is in its directory. The dependency between other packages is defined in sfdx-project.json file",
+            "line": 1
+        },
+        {
+            "file": "sfdx-project.json",
+            "description": "All the custom objects, custom fields, layouts, tabs, and permission sets for objects are contained in the es-base-objects directory",
+            "line": 4
+        },
+        {
+            "file": "sfdx-project.json",
+            "description": "es-base-styles directory holds all the styling related lightning web components, logos, themes, and  lightning app page template metadata",
+            "line": 11
+        },
+        {
+            "file": "sfdx-project.json",
+            "line": 18,
+            "description": "es-base-code directory contains code components that are reusable. This directory has custom metadata definitions and records along with the Lightning Messaging Service (LMS) metadata"
+        },
+        {
+            "file": "sfdx-project.json",
+            "description": "es-space-mgmt directory contains most of the metadata including the Lightning Console app, lightning web components, flows, lightning pages and other components required for the Easy Spaces application",
+            "line": 31
+        }
+    ]
+}
```

Comment:
I would add the steps at the folder level instead of inside the sfdx-project.json. Its best if we get into this file in the "Unlocked Packages" tour.

Reply:
Also please check the case of the feature titles like LWC etc.

Reply:
fixed!